samples/RenderingPlayground/RenderingPlayground.csproj

```diff
@@ -1,10 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <LangVersion>latest</LangVersion>
+    <!--<StartupObject>RenderingPlayground.Program</StartupObject>-->
```

Comment:
I like this approach but I would propose having a common name between MSBuild and the C#. Perhaps `EntryPointFullTypeName`?

Reply:
Kindly, do you mean - " ... propose __NOT_ having a common name" ?
I was also concerned that it may cause problems to `AutoGeneratedProgram.Main()` bootstrap entry point but after testing `<StartupObject>` for a while I observed no any issues with it. So, I think we should give it a chance and if something goes wrong change it to `EntryPointFullTypeName`.  What do you think?

Reply:
I meant have this name match the parameter name being passed into `ExecuteAssemblyAsync`. The casing should obviously be different between them, but I believe the names should match. 
If someone were to enter a type name that did not exist, they would get the error `$"Could not find entry point class '{entryPoint}'."`. This exception message makes sense in the context of the `ExecuteAssemblyAsync` method, but does not indicate that the issue is the `StartupObject` inside of the project file. If these names matched it would be more obvious where the incorrect value were specified.